### DISCUSSION TITLE
Gmail SMTP configuration [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2430] [Discord Threads]
> 🙋‍♂️ **Ra**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Thanks <@961503950778023956> for bringing this up. We will explore further on this.  
#  
> 👨‍🦲 **dstala**  
#### <@961503950778023956> , I was able to successfully configure GMail SMTP using app-password. Details here https://www.youtube.com/watch?v=rpmfDHCyPbo  
#  
> 👨‍🦲 **dstala**  
#### Ping me if you get stuck or need any other help  
#  
> 🙋‍♂️ **Ra**  
#### Thank you! I will try it out  
#  
> 🙋‍♂️ **Ra**  
#### I watched the video but it still seems like a hack / workaround. I will use it if I need to but is there any other straightforward alternative to use emails?  
#  
> 👨‍🦲 **dstala**  
#### <@961503950778023956> right now, app-password is only way for Gmail SMTP. it's one time configuration and should be secure enough - as original gmail credentials are not exposed at all. we will consider expanding it. do help us with a feature request here.    
#### https://github.com/nocodb/nocodb/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature%5D+  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984588480959094887)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
